src/main/java/org/isf/lab/service/LabIoOperations.java

```diff
@@ -76,6 +79,27 @@ public List<LaboratoryRow> getLabRow(Integer code) throws OHServiceException {
 		return rowRepository.findByLaboratory_Code(code);
 	}
 
+	/**
+	 * Return the list of exams ({@link Laboratory}s) divided by pages.
+	 * 
+	 * @param oneWeek
+	 * @param pageNo
+	 * @param pageSize
+	 * @return the list of {@link Laboratory}s (could be empty)
+	 * @throws OHServiceException
+	 */
+	public List<Laboratory> getLaboratory(boolean onWeek, int pageNo, int pageSize) throws OHServiceException {
```

Comment:
The Javadoc has the variable as  `oneWeek` (see line 85) and the method signature uses `onWeek`; pick one